### PR TITLE
Update Helm release topolvm to v11

### PR DESCRIPTION
**Pull request by vshn-renovate**
This PR contains the following updates:

| Package | Update | Change |
|---|---|---|
| [topolvm](https://togithub.com/topolvm/topolvm) | major | `10.0.0` -> `11.0.0` |

---

### Configuration

📅 **Schedule**: Branch creation - At any time (no schedule defined), Automerge - At any time (no schedule defined).

🚦 **Automerge**: Disabled by config. Please merge this manually once you are satisfied.

♻ **Rebasing**: Renovate will not automatically rebase this PR, because other commits have been found.

🔕 **Ignore**: Close this PR and you won't be reminded about this update again.

---

 - [ ] If you want to rebase/retry this PR, click this checkbox. ⚠ **Warning**: custom changes will be lost.

---

This PR has been generated by [Renovate Bot](https://togithub.com/renovatebot/renovate).

### CODE REVIEW

**Comment by DebakelOrakel** [comment]
Caution: https://github.com/topolvm/topolvm/blob/main/CHANGELOG.md#0160---2022-12-05 (!)

**Comment by simu** [comment]
> Caution: https://github.com/topolvm/topolvm/blob/main/CHANGELOG.md#0160---2022-12-05 (!)

TL;DR: Make this upgrade a breaking change, and strongly recommend to set `helm_values.useLegacy: true` for existing setups in the major update migration guide, but let the component setup TopoLVM with the `topolvm.io` apiGroup for new installations.

File: Makefile.vars.mk [hunk 1]
```diff
@@ -50,4 +50,4 @@ KUBENT_IMAGE    ?= ghcr.io/doitintl/kube-no-trouble:latest
 KUBENT_DOCKER   ?= $(DOCKER_CMD) $(DOCKER_ARGS) $(root_volume) --entrypoint=/app/kubent $(KUBENT_IMAGE)
 
 instance ?= defaults
-test_instances = tests/defaults.yml tests/full.yml
+test_instances = tests/defaults.yml tests/legacy.yml
```
Comment (simu) [inline Makefile.vars.mk:53]: Generally, you should add additional tests with `commodore component update -t <testName> .`. That way, you'll ensure that all the component boilerplate (GitHub actions, Makefile, etc.) are updated to know about the new test case.

File: docs/modules/ROOT/pages/how-tos/upgrade-v3-to-v4.adoc [hunk 2]
```diff
@@ -0,0 +1,16 @@
+= Upgrade from v3 to v4
+
+This release contains a provisioner name change from `topolvm.cybozu.com` to `topolvm.io`.
+
+If you update from v3 you need to enable `useLegacy` parameter.
```
Comment (simu) [inline docs/modules/ROOT/pages/how-tos/upgrade-v3-to-v4.adoc]:
```suggestion
If you update from v3, you need to enable `useLegacy` parameter.
```

**Review by simu — COMMENTED** [review]
Looks like the `legacy` test case is missing in the GitHub actions config. I'd recommend using `commodore component update -t legacy .` to ensure the component boilerplate is updated correctly when adding a new test.

File: .cruft.json [hunk 3]
```diff
@@ -7,14 +7,14 @@
...
-      "test_cases": "defaults",
+      "test_cases": "defaults legacy",
       "add_lib": "n",
       "add_pp": "n",
       "add_golden": "y",
       "add_matrix": "y",
       "add_go_unit": "n",
       "copyright_holder": "VSHN AG <info@vshn.ch>",
-      "copyright_year": "2022",
+      "copyright_year": "2023",
```
Comment (simu) [inline .cruft.json:17]: You'll probably want to adjust this to 2021 so that the Cruft config matches the `LICENSE` file

```suggestion
      "copyright_year": "2021",
```

**Review by simu — APPROVED** [review]
